mnefun/_mnefun.py

```diff
@@ -1725,6 +1725,36 @@ def gen_forwards(p, subjects, structurals, run_indices):
             fwd = make_forward_solution(
                 info, trans, src, bem, n_jobs=p.n_jobs, mindist=p.fwd_mindist)
             write_forward_solution(fwd_name, fwd, overwrite=True)
+        # plot BEM & source space alignment
+        plt.rcParams['axes.titlesize'] = 'medium'
+        plt.rcParams['axes.titleweight'] = 'normal'
```

Comment:
This will make permanent changes (bad), use:

https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.rc_context.html

Or just tweak the axes or object later (probably easiest)